docs/whatsnew/4.1.rst

```diff
@@ -33,6 +33,18 @@ between the consecutive coordinates, in pixel space, and then indexing the data
 array of the map at those points.
 See :ref:`sphx_glr_generated_gallery_units_and_coordinates_map_slit_extraction.py` for an example.
 
+Drawing the solar equator and prime meridian
+============================================
+You can now use the `~sunpy.visualization.drawing` module to draw the solar equator
+and prime meridian (zero Carrington longitude) on Astropy WCS axes. See
+:ref:`plot_equator_prime_meridian.py` for an example.
```

Comment:
Same as the changelog, if we can ref the gallery example directly.

Reply:
In the [What's New 3.1](https://docs.sunpy.org/en/stable/whatsnew/3.1.html) page (for example), there are several examples that are referenced using a fancy-looking "minigallery" command. Is that method of referencing preferred in the What's New documentation?